NOMENCLATURE.md

```diff
@@ -72,46 +72,47 @@ maintains a connection to the _dispatcher_, waiting for the current set of
 tasks assigned to the node. Assigned tasks are then dispatched to the engine.
 The agent notifies the _dispatcher_ of the current state of assigned tasks.
 
-This is roughly analagous to an entertainment agent. It ensures the worker has
-the correct set of work and let's others know what the worker is doing.
+This is roughly analagous to an entertainment (What is this?) agent. It ensures
```

Comment:
What do you mean by "What is this?".

I think it's a joke but I'm pretty dense and it didn't click for me.